guides/common/Makefile

```diff
@@ -61,7 +61,7 @@ $(DEST_DIR)/$(BUILD).css: $(BUILD_DIR)/default.css
 
 $(IMAGES_TS): $(IMAGES)
 	cp $? $(IMAGES_DIR)/
-	pushd "$(IMAGES_DIR)" >/dev/null && for IMG in *.*; do ln -s ../../images/$$IMG ../common/images/$$IMG; done && popd >/dev/null
+	pushd "$(IMAGES_DIR)" >/dev/null && for IMG in *.*; do ln -fs ../../images/$$IMG ../common/images/$$IMG; done && popd >/dev/null
```

Comment:
@lzap what's happening here?

Reply:
Oh that is just adding the force option - in some rare cases Makefile would attempt to copy over existing files and errors out. This change makes it to just overwrite them. It does not do this always, just in edge cases.